src/Microsoft.Extensions.PlatformAbstractions.Internal/Native/NativeMethods.Unix.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Extensions.PlatformAbstractions.Internal.Native
+{
+    internal static partial class NativeMethods
+    {
+        public static class Unix
+        {
+            public unsafe static string GetUname()
```

Comment:
I do believe this is exposed by `System.Runtime.InteropServices.RuntimeInformation.OSDescription`.  Maybe we could use that instead?

https://github.com/dotnet/corefx/blob/master/src/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.Unix.cs#L25

calls

https://github.com/dotnet/corefx/blob/master/src/Native/System.Native/pal_runtimeinformation.cpp#L16